lib/core/sidenav-layout/components/sidenav-layout/sidenav-layout.component.ts

```diff
@@ -38,6 +38,8 @@ export class SidenavLayoutComponent implements OnInit, AfterViewInit, OnDestroy
     @Input() hideSidenav = false;
     @Input() expandedSidenav = true;
 
+    @Output() notify = new EventEmitter();
```

Comment:
Generic type of the EventEmitter is missing. I suppose it would be EventEmitter<boolean>. Instead of this general "notify" name, what about using a more meaningful name, like "expanded".